moskito-backend/src/Serializer/CampsiteSerializer.php

```diff
@@ -0,0 +1,52 @@
+<?php 
+
+namespace App\Serializer;
+
+use App\Entity\Campsite;
+
+class CampsiteSerializer {
+    public function serialize($elements){
+        $Json = [];
+        if (is_array($elements)) {
+            foreach($elements as $element) {
+                $Json[] = [
```

Comment:
Variable name $Json might be semantically misleading. Maybe $elementasArray or smth similar?